Detectors/CTP/reconstruction/src/RawDataDecoder.cxx

```diff
@@ -390,18 +315,26 @@ int RawDataDecoder::decodeRaw(o2::framework::InputRecord& inputs, std::vector<o2
       }
       nwrites++;
     }
-    mStickyError = false;
     // LOG(error) << "CTP decoding IR errors:" << mErrorIR << " TCR errors:" << mErrorTCR;
   }
   return ret;
 }
 //
+int RawDataDecoder::decodeRaw(o2::framework::InputRecord& inputs, std::vector<o2::framework::InputSpec>& filter, std::vector<CTPDigit>& digits, std::vector<LumiInfo>& lumiPointsHBF1)
+{
+  o2::pmr::vector<CTPDigit> pmrdigits;
+  for (auto const d : digits) {
+    pmrdigits.push_back(d);
+  }
+  return decodeRaw(inputs, filter, pmrdigits, lumiPointsHBF1);
```

Comment:
I don't understand this code, I guess 1st you wanted to decode digits one type of vectors then copy it to another, but why do you need to copy at all, use template and fill the final digits vector directly.

Reply:
This want to meant to be temporary decode( ...std::vector<CTPDigit>& digits...) needed for QC
/sw/SOURCES/QualityControl/v1.114.2/v1.114.2/Modules/CTP/src/RawDataQcTask.cxx:73:44: error ...
Later RawDataQcTask.cxx can be changed to o2::pmr::vector ?

Reply:
But you are calling decode after filling the pmrdigits by the values from the digits. Where these latter are filled?

Reply:
yes, you are right, I did opposite what I wanted. Fixing.

Reply:
This one is ok as it is in github ?

Reply:
no